bitnami/vault/values.yaml

```
@@ -1387,6 +1387,18 @@ injector:
   ## @param injector.customStartupProbe Custom startupProbe that overrides the default one
   ##
   customStartupProbe: {}
+
+  ## @param injector.webhook Enabling specify which namespace you want to work with webhook
+  ## Example:
+  ## webhook:
+  ##   namespaceSelector:
+  ##     matchExpressions:
+  ##     - key: kubernetes.io/metadata.name
+  ##       operator: In
+  ##       values:
+  ##         - "{{ .Release.Namespace }}"
+  webhook: {}
+       
```

Please use the following structure:

```suggestion
  webhook: 
    ## @param injector.webhook Enabling specify which namespace you want to work with webhook
    ## Example:
    ## webhook:
    ##   namespaceSelector:
    ##     matchExpressions:
    ##     - key: kubernetes.io/metadata.name
    ##       operator: In
    ##       values:
    ##         - "{{ .Release.Namespace }}"
    namespaceSelector: {}
```

This way users should not be confused about what `webhook.*` values can be configured or not, and leaves room for new values to be added in the future.

---

Someting failed during the creation of the Readme.md
Do you know why @migruiz4 ?

---

My bad, I fixed it in the suggestion above.

The README is autogenerated based on the `@param` tag, which was missing `.namespaceSelector`.
It should be `@param injector.webhook.namespaceSelector`